### ISSUE TITLE
Add a custom Reline color scheme to the `Core::CLI::RubyShell`

### ISSUE THREAD

**Issue by postmodern** [ronin-rb/ronin-core#44]
As of `reline` 0.4.0, reline now supports [custom color schemes](https://ydah.net/blog/posts/20240212/) for the tab-completion menu. Ronin could customize the default cyan background + grey text + magenta highlighted text completion menu.

**Comment by moozzi** [comment]
![Screenshot 2024-08-31 at 13 38 48](https://github.com/user-attachments/assets/aac4c3fa-dacc-4ca9-bcd8-e1da02c353b0)

Something like that?

**Comment by postmodern** [comment]
@AI-Mozi nice! We should probably think about the color scheme.

**Comment by moozzi** [comment]
Oh.. I thought you want a `cyan background + grey text + magenta highlighted text` 😅
I'll play with it a bit, but I think the black/white/gray/navy color pallet is associated with ronin.

**Comment by moozzi** [comment]
![Screenshot 2024-09-01 at 18 19 16](https://github.com/user-attachments/assets/1be87dd8-dad9-4c30-a6f0-1247e4687165)

@postmodern  what do you think?
I like that red background for highlighted option as it matches the `irb(ronin-test)>`

**Comment by postmodern** [comment]
@AI-Mozi I played around with some different color schemes. Here's white text on gray:

```ruby
          Reline::Face.config(:completion_dialog) do |conf|
            conf.define :default, foreground: :white, background: :gray
            conf.define :enhanced, foreground: :black, background: :white
            conf.define :scrollbar, foreground: :white, background: :gray
          end
```

![ronin_ruby_shell_white_on_gray](https://github.com/user-attachments/assets/ce4f4bac-23d8-498a-acc7-cfd761b9b7f5)

and here is white text on black:

```ruby
          Reline::Face.config(:completion_dialog) do |conf|
            conf.define :default, foreground: :white, background: :black
            conf.define :enhanced, foreground: :black, background: :white
            conf.define :scrollbar, foreground: :white, background: :black
          end
```

![ronin_ruby_shell_white_on_black](https://github.com/user-attachments/assets/9a6c0087-caf8-4a88-a6c3-efc6b53e00ee)

Here's what white on gray with red highlighting looks like in my terminal:
![ronin_ruby_shell_white_on_gray_with_red](https://github.com/user-attachments/assets/2e7a7f23-f61b-4785-acee-781ace6771c8)

Thoughts on which one is better?

**Comment by moozzi** [comment]
`white on gray with red highlighting` does not look as good as in my terminal..
Maybe just `white text on gray`?

**Comment by postmodern** [comment]
@AI-Mozi yeah I'm liking both white text on gray and white text on black.

**Comment by moozzi** [comment]
@postmodern I'll check which one looks better in my terminal, and I'll chose it

**Comment by postmodern** [comment]
Implemented by PR #57.
